index.html

```diff
@@ -158,70 +165,74 @@ <h3 class="sm-text-2">A daily selection of privately personalized reads; no acco
-          </p>
-          <h3 class="lg-text-3">Let's connect</h3>
-          <ul class="social-media-2">
-            <li><img src="./assets/cat.svg" alt="icon" /></li>
-            <li><img src="./assets/linkedin.svg" alt="icon" /></li>
-            <li><img src="./assets/victory.svg" alt="icon" /></li>
-            <li><img src="./assets/twitter.svg" alt="icon" /></li>
-            <li><img src="./assets/M.svg" alt="icon" /></li>
-          </ul>
-          <button class="btn actions" type="button">Get my resume</button>
-        </div>
-        <div class="skill-list">
-          <div class="languages">
-            <h3 class="sm-text-2">Languages</h3>
-          </div>
-          <ul class="language-list">
-            <li><img src="./assets/javascript.svg" alt="" />JavaScript</li>
-            <li><img src="./assets/html.svg" alt="" />HTML</li>
-            <li><img src="./assets/css.svg" alt="" />CSS</li>
-          </ul>
-          <div class="frameworks">
-            <h3 class="sm-text-2">Frameworks</h3>
+      <section>
+        <div class="div-container about">
+          <div class="top">
+            <h2 class="lg-text-1 text-width">About Myself</h2>
+            <p class="sm-text-1">
+              Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+              experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <h3 class="lg-text-3">Let's connect</h3>
+            <ul class="social-media-2">
+              <li><img src="./assets/cat.svg" alt="icon" /></li>
+              <li><img src="./assets/linkedin.svg" alt="icon" /></li>
+              <li><img src="./assets/victory.svg" alt="icon" /></li>
+              <li><img src="./assets/twitter.svg" alt="icon" /></li>
+              <li><img src="./assets/M.svg" alt="icon" /></li>
+            </ul>
+            <button class="btn actions" type="button">Get my resume</button>
           </div>
-          <hr />
-          <div class="skills">
-            <h3 class="sm-text-2">Skills</h3>
+          <div class="skill-list">
+            <div class="languages">
+              <h3 class="sm-text-2">Languages</h3>
+            </div>
+            <ul class="language-list">
+              <li><img src="./assets/javascript.svg" alt="" />JavaScript</li>
+              <li><img src="./assets/html.svg" alt="" />HTML</li>
+              <li><img src="./assets/css.svg" alt="" />CSS</li>
+            </ul>
+            <div class="frameworks">
+              <h3 class="sm-text-2">Frameworks</h3>
+            </div>
+            <hr />
+            <div class="skills">
+              <h3 class="sm-text-2">Skills</h3>
+            </div>
+            <hr />
           </div>
-          <hr />
         </div>
-
-        <section class="contact-me">
+      </section>
+      <section class="bg-color-1">
```

Comment:
- [ ] As this is the last part of the web page, It is nice to use the `footer` tag instead of the `section` tag. KIndly replace it with the `footer` tag.